plot_app/configured_plots.py

```diff
@@ -141,7 +141,82 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page):
                  bokeh_plot=data_plot.bokeh_plot)
         if data_plot.finalize() is not None:
             plots.append(data_plot.bokeh_plot)
-            if not is_running_locally(): # do not enable Google Map if running locally
+            # Position focus
+            try:
+                def rgb_colors(flight_mode):
+                    """ flight mode colors for KML file """
+                    if not flight_mode in flight_modes_table: flight_mode = 0
+
+                    color_str = flight_modes_table[flight_mode][1][1:] # color in form 'ff00aa'
+
+                    # increase brightness to match colors with template
+                    rgb = [int(color_str[2*x:2*x+2], 16) for x in range(3)]
+                    for i in range(3):
+                        rgb[i] += 40
+                        if rgb[i] > 255: rgb[i] = 255
+
+                    color_str = "".join(map(lambda x: format(x, '02x'), rgb))
+
+                    return '#'+color_str
+
+                def ulog_to_polyline(ulog):
+                    # get flight modes
```

Comment:
Can you use the variable from https://github.com/PX4/flight_review/blob/master/plot_app/configured_plots.py#L45, instead of recomputing it?